src/main/java/lotto/domain/AutoLottos.java

```diff
@@ -0,0 +1,27 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class AutoLottos {
+    private final List<Lotto> lottos = new ArrayList<>();
+
+    public AutoLottos() {
+    }
+
+    public void createAutoLottos(int lottoCount, Generator generator) {
+        while (lottoCount-- > 0) {
+            lottos.add(getNewLotto(generator));
+        }
+    }
```

Comment:
```suggestion
        while (lottoCount > 0) {
            lottos.add(getNewLotto(generator));
            lottoCount--;
        }
    }
```
개인적으론 이게 더 직관적이지 않나 싶어요 ! 지극히 개인 취향..